workspaces/github-actions/plugins/github-actions/README.md

```diff
@@ -140,8 +162,6 @@ integrations:
 ## Limitations
 
 - There is a limit of 100 apps for one OAuth client/token pair
-- The OAuth application must be at the GitHub organization level in order to display the workflows. If you do
```

Comment:
nit/don't have to change - but would maybe add one disclaimer here (or higher up under the "provide Oauth credentials" area) starting that using org level OAuth instead of one associated with a user will limit which repositories are functional...should be obvious but can imagine someone not realizing that and getting confused

Reply:
Done!